doc/source/user/quickstart.rst

```diff
@@ -1112,8 +1112,8 @@ indices for each dimension must have the same shape.
            [[10,  9],
             [11, 11]]])
 
-Naturally, we can put ``i`` and ``j`` in a sequence (say a list) and
-then do the indexing with the list.
+Naturally, we can put ``i`` and ``j`` in a sequence (say a tuple) and
+then do the indexing with the tuple.
```

Comment:
Perhaps:
```suggestion
In Python, putting multiple indices within ``[]`` like this is exactly the same as putting ``i`` and ``j`` in a ``tuple``, and
then doing the indexing with that.
```
Let's see if anyone else has wording suggestions before changing that

Reply:
Yes that is indeed a more naturally flowing sentence